pypeman/helpers/lazyload.py

```diff
@@ -0,0 +1,52 @@
+import importlib
+import traceback
+
+def load_class(module, class_, deps):
+    """
+    Try to load a class if all deps can be imported.
+    :param module: path to module contains class
+    :param class_:
+    :param deps:
+    :return:
+    """
+    try:
+        mod = importlib.import_module(module)
+        return getattr(mod, class_)
+    except ImportError as exc:
+        traceback.print_exc()
+        msg = str(exc)
+
+        # Try to find any dependency in message
+        found = False
+        for dep in deps:
+            if dep in msg:
+                found = True
+                break
+
+        if not found:
+            print("IMPORT ERROR...")
+            raise
+
+        print("%s module not activated" % module)
```

Comment:
all error prints should be to stderr not not be mixed with stdout.
that's kind of common practice for most tools